### ISSUE TITLE
Add (docker) compose

### ISSUE THREAD

**Issue by avra911** [alexpunct/chatgpt-journal#23]
**Docker compose support**

Docker Compose is a tool that is useful for managing and orchestrating multi-container Docker applications. It allows you to define your application’s services, networks, and volumes in a YAML file and then run and scale the entire application with a single command.

**Comment by alexpunct** [comment]
you don't really need Docker compose. If you want to run Supabase locally, they have a Docker compose with all the services. Or you can use their cloud hosted free service. You only need to run the application

**Comment by avra911** [comment]
> you don't really need Docker compose. If you want to run Supabase locally, they have a Docker compose with all the services. Or you can use their cloud hosted free service. You only need to run the application

I prefer to have every application in docker, is easier for onboarding anyone from any operating system. For example I dont have npm installed in my system and to deal with different tag/versions locally.

**Comment by alexpunct** [comment]
You are right, I meant the docker compose for the whole setup. But for running the app itself, a Dockerfile would be nice indeed. Feel free to send a PR. And if you want to send a docker-compose that includes Supabase too, that would be even better :)

**Comment by avra911** [comment]
Thanks.

I actually searched if you can run in linux npm 0.0.1 alongside with 1.0.0 and I found a way to switch but have not find a way to use both for 2 different apps to run on the same time.
